File: src/org/elixir_lang/refactoring/module_attribute/rename/Processor.java
package org.elixir_lang.refactoring.module_attribute.rename;

import com.intellij.psi.PsiElement;
import com.intellij.refactoring.rename.RenamePsiElementProcessor;
import org.jetbrains.annotations.NotNull;

import static org.elixir_lang.refactoring.module_attribute.rename.Handler.isAvailable;

public class Processor extends RenamePsiElementProcessor {
    @Override
    public boolean canProcessElement(@NotNull PsiElement element) {
        return isAvailable(element);
    }
}

File: src/org/elixir_lang/refactoring/module_attribute/rename/Processor.kt
package org.elixir_lang.refactoring.module_attribute.rename

import com.intellij.psi.PsiElement
import com.intellij.refactoring.rename.RenamePsiElementProcessor
import org.elixir_lang.refactoring.module_attribute.rename.Handler.Companion.isAvailable

class Processor : RenamePsiElementProcessor() {
    override fun canProcessElement(element: PsiElement): Boolean = isAvailable(element)
}
